indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -398,9 +417,21 @@ def _checkParams(self, params):
                               .all()) if ids else []
         self.template_id = request.args.get('template_id')
 
+    def _get_format(self, tpl):
+        pixels_cm = 50
```

Comment:
I believe there is a better way to retrieve this. We could reuse the same from `pdf.py`